modules/tensor_mechanics/tests/cp_slip_rate_integ/crysp_sinteg.i

```diff
@@ -0,0 +1,218 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 3
+  nx = 2
+  ny = 2
+  nz = 2
+  elem_type = HEX8
+  displacements = 'ux uy uz'
+[]
+
+[Variables]
+  [./ux]
+    block = 0
+  [../]
+  [./uy]
+    block = 0
+  [../]
+  [./uz]
+    block = 0
+  [../]
+[]
+
+[AuxVariables]
+  [./stress_zz]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+  [./fp_zz]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+  [./e_zz]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+  [./gss1]
+    order = CONSTANT
+    family = MONOMIAL
+    block = 0
+  [../]
+[]
+
+[Functions]
+  [./tdisp]
+    type = ParsedFunction
+    value = 0.01*t
+  [../]
+[]
+
+[Kernels]
+  [./TensorMechanics]
+    disp_z = uz
+    disp_y = uy
+    disp_x = ux
+    use_displaced_mesh = true
+  [../]
+[]
+
+[AuxKernels]
+  [./stress_zz]
+    type = RankTwoAux
+    variable = stress_zz
+    rank_two_tensor = stress
+    index_j = 2
+    index_i = 2
+    execute_on = timestep_end
+    block = 0
+  [../]
+  [./fp_zz]
+    type = RankTwoAux
+    variable = fp_zz
+    rank_two_tensor = fp
+    index_j = 2
+    index_i = 2
+    execute_on = timestep_end
+    block = 0
+  [../]
+  [./e_zz]
+    type = RankTwoAux
+    variable = e_zz
+    rank_two_tensor = lage
+    index_j = 2
+    index_i = 2
+    execute_on = timestep_end
+    block = 0
+  [../]
+  [./gss1]
+    type = MaterialStdVectorAux
+    variable = gss1
+    property = gss
+    index = 0
+    execute_on = timestep_end
+    block = 0
+  [../]
+[]
+
+[BCs]
+  [./symmy]
+    type = PresetBC
+    variable = uy
+    boundary = bottom
+    value = 0
+  [../]
+  [./symmx]
+    type = PresetBC
+    variable = ux
+    boundary = left
+    value = 0
+  [../]
+  [./symmz]
+    type = PresetBC
+    variable = uz
+    boundary = back
+    value = 0
+  [../]
+  [./tdisp]
+    type = FunctionPresetBC
+    variable = uz
+    boundary = front
+    function = tdisp
+  [../]
+[]
+
+[Materials]
+  active = 'crysp'
+  [./crysp]
+    type = FiniteStrainCrystalPlasticity
+    block = 0
+    disp_y = uy
+    disp_x = ux
+    gtol = 1e-2
+    slip_sys_file_name = input_slip_sys.txt
+    disp_z = uz
+    C_ijkl = '1.684e5 1.214e5 1.214e5 1.684e5 1.214e5 1.684e5 0.754e5 0.754e5 0.754e5'
+    nss = 12
+    num_slip_sys_flowrate_props = 2 #Number of properties in a slip system
+    flowprops = '1 4 0.001 0.1 5 8 0.001 0.1 9 12 0.001 0.1'
+    hprops = '1.0 541.5 60.8 109.8 2.5'
+    gprops = '1 4 60.8 5 8 60.8 9 12 60.8'
+    fill_method = symmetric9
+    tan_mod_type = exact
+  [../]
+  [./elastic]
+    type = FiniteStrainElasticMaterial
```

Comment:
This uses the outdated material system that is based on TensormechanicsMaterial. We are moving to a new system where we have separate material classes for stress, strain, elasticity tensors etc.